src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Shared/_FlashMessagePartial.cshtml

```diff
@@ -2,27 +2,10 @@
 @if(TempData.ContainsKey(ITempDataDictionaryExtensions.FlashMessageTempDataKey))
 {
     var message = TempData[ITempDataDictionaryExtensions.FlashMessageTempDataKey];
-    var level = (ITempDataDictionaryExtensions.FlashMessageLevel) TempData[ITempDataDictionaryExtensions.FlashMessageLevelTempDataKey];
-    
-<div class="govuk-grid-column-two-thirds das-notification govuk-!-margin-bottom-4 das-notification--@GetCssClassForLevel(level)" tabindex="-1">    
-    <h3 class="das-notification__heading">@message</h3>
-</div>
-}
 
-@functions
-{
-    private string GetCssClassForLevel(ITempDataDictionaryExtensions.FlashMessageLevel level)
-    {
-        switch(level)
-        {
-            case ITempDataDictionaryExtensions.FlashMessageLevel.Info:
-                return "info";
-            case ITempDataDictionaryExtensions.FlashMessageLevel.Warning:
-                return "warning";
-            case ITempDataDictionaryExtensions.FlashMessageLevel.Success:
-                return "success";
-            default:
-                throw new ArgumentOutOfRangeException(nameof(level));
-        }
-    }
-}
\ No newline at end of file
+    <div class="govuk-panel govuk-panel--confirmation">
```

Comment:
this is going to conflict with changes made here: 

https://github.com/SkillsFundingAgency/das-employercommitments-v2/pull/204

I think the linked code should be used because some flash messages use the info tag